### PR TITLE
consistency in README.md

### PR DESCRIPTION
**Pull request by Korfor**
- code blocks should have a language specified
- Headings should be surrounded by blank lines
- Since only python3 is supported changed `pip` to `pip3` and `python` to `python3`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +14,13 @@ Inspired from package [profanity](https://github.com/ben174/profanity) of [Ben F
...
 ## Requirements
+
 This package works with `Python 3.4+` and `PyPy3`.
 
 ## Installation
-```
-$ pip install better_profanity
+
+```sh
+$ pip3 install better_profanity
```
Comment (snguyenthanh) [inline README.md:23]: Remove the `$` symbol.

**Review by snguyenthanh — CHANGES_REQUESTED** [review]
Thanks for the PR. I think only 1 small change is required.

File: README.md (hunk 1 above)
Comment (Korfor) [reply README.md:23]: its fine by me. I didn't put it there

**Review by snguyenthanh — APPROVED** [review]

**Comment by Korfor** [comment]
Thanks! Have a great one

**Pull request merged**